2023-06-30 00:55:16,248 - logger_ser - ERROR - app.ocr_type_document():76 - Id not found!
2023-06-30 00:55:16,249 - logger_ser - ERROR - app.ocr_type_document():86 - 
Traceback (most recent call last):
  File "/media/HDD1/hai/api-server-template/server/app.py", line 77, in ocr_type_document
    raise Exception
Exception
2023-06-30 00:55:19,629 - logger_ser - ERROR - app.ocr_type_document():76 - Id not found!
2023-06-30 00:55:19,630 - logger_ser - ERROR - app.ocr_type_document():86 - 
Traceback (most recent call last):
  File "/media/HDD1/hai/api-server-template/server/app.py", line 77, in ocr_type_document
    raise Exception
Exception
2023-06-30 00:55:22,671 - logger_ser -  INFO - app.__handle_request():49 - Success!
2023-06-30 00:55:56,023 - logger_ser -  INFO - app.__handle_request():49 - Success!
2023-06-30 00:56:04,339 - logger_ser -  INFO - app.__handle_request():49 - Success!
